skycultures/arabian_peninsula/description.en.utf8

```diff
@@ -0,0 +1,109 @@
+</p>
+<h4 id="sources-of-star-names-and-asterisms">Sources of Star Names and Asterisms</h4>
+<p>The source of this sky culture is a compilation of star names from oral traditions and the following books:</p>
+<ol>
+<li>Al-'Oboodi, Mohammad b. Nassir, The folk proverbs in Najd, 4 volumes, Dar al-Thuloothiyah, Riyadh, Saudi Arabia, 2010 AD.
+<br>محمد بن ناصر العبودي، الأمثال العامية في نجد، دار الثلوثية، الرياض، المملكة العربية السعودية، 1431 هـ.</li>
+<li>Al-'Oboodi, Mohammad b. Nassir, Lexicon of the classical root of the common dialect words, 13 volumes, King AbdulAzeez Library, Riyadh,  Saudi Arabia, 2009 AD.
+<br>محمد بن ناصر العبودي، معجم الأصول الفصيحة للألفاظ الدارجة، مكتبة الملك عبد العزيز، الرياض، المملكة العربية السعودية، 1430 هـ.</li>
+<li>A collection of Bedouins poetry by the name: The moist flowers of Bedouin Poetry, 18 volumes, Al Ma'arif library, Tayef, Saudi Arabia.
+<br>الأزهار النادية من أشعار البادية، مكتبة المعارف، الطائف، المملكة العربية السعودية.</li>
+<li>A collection of poetry by Mandeel al-Fuhaid, Our local literature of the Arab peninsula, 10 volumes, Riyadh, Saudi Arabia.
+<br>منديل بن محمد الفهيد، من آدابنا الشعبية في الجزيرة العربية.</li>
+<li>AbdulRahman al-Suwaida, The local common proverbs of the Hayel region, Riyadh,  Saudi Arabia, 2007.
+<br>عبد الرحمن بن زيد السويداء، الأمثال الشعبية السائرة في منطقة حائل، الرياض، المملكة العربية السعودية، 1428 هـ.</li>
+<li>AbdulRahman al-Suwaida, The best of folk poetry, 3 volume Poetry collection, Riyadh,  Saudi Arabia, 1999.
+<br>عبد الرحمن بن زيد السويداء، درر من الشعر الشعبي، الرياض، المملكة العربية السعودية، 1420 هـ.</li>
+<li>AbdulRahman al-Suwaida, The Poets of the Mountain, 5 volume Poetry collection, Riyadh,  Saudi Arabia, 2013.
+<br>عبد الرحمن بن زيد السويداء، شعراء الجبل الشعبيون، الرياض، المملكة العربية السعودية، 1434 هـ.</li>
+<li>AlAjaji, Khalid, The Explanation of al-Qadhi Poem Anwa'a and Stars, Riyadh,  Saudi Arabia, 2013.
+<br>خالد بن عبد الله العجاجي، شرح قصيدة القاضي في الأنواء والنجوم، الرياض، المملكة العربية السعودية، 1434 هـ.</li>
+<li>AlAjaji, Khalid, The Explanation of al-Khalawi Astronomy Poems, Riyadh,  Saudi arabia, 2018.
+<br>خالد بن عبد الله العجاجي، شرح مأثورات الخلاوي الفلكية، الرياض، المملكة العربية السعودية، 1439 هـ.</li>
+</ol>
+<h2 id="constellations">Constellations</h2>
+<h5 id="al-nayem-the-seven-daughters-of-na-sh">Al-Nayem - The Seven - Daughters of Na'sh</h5>
+<p>(Arabic النعايم), meaning is not known for certain, composed of the seven stars of the Big Dipper. other names are: <em>The Seven</em> (Arabic السبَّع) and <em>Daughters of Na'sh</em> (Arabic بنات نعش). The first name is common in oral poetry.</p>

```

Comment:
Can you please add an English transliteration to the Arabic phrases? It helps most interested people reading the original words. I think some spelling like 

`(Arabic النعايم <it>al-nayem</it>)`

would be nice. (I never can read the loop-like letter between n and a!)

Reply:
We can do this for the names in all Arabic sky cultures. The difficulty is that using one of the standards for transliterations needs some learning by the reader. For example:

```(Arabic النعايم _an-Naʿayem_)```

Using ʿ to indicate the letter (عـ) in Arabic. ʿ or sometimes superscript c is similar in shape to the Arabic letter, but far from how that letter is pronounced. This is used by prof. Paul Kunitzsch and Roland Laffitte.

This is the standard I used when working with Laffitte:

| Arabic letter | Transliteration | Arabic letter | Transliteration | Arabic letter | Transliteration |
|---------------|-----------------|---------------|-----------------|---------------|-----------------|
| ء | ‘ | ر | r | ف | f |
| ا | ā | ز | z | ق | q |
| ب | b | س | s | ك | k |
| ت | t | ش | š | ل | l |
| ث | ṯ | ص | ṣ | م | m |
| ج | ğ | ض | ḍ | ن | n |
| ح | ḥ | ط | ṭ | ه | h |
| خ | ḫ | ظ | ẓ | و | w |
| د | d | ع | c | ي | y |
| ذ | ḏ | غ | ġ |   |   |

Short vowels : a, i, u
Long vowels : ā, ī, ū

transliteration of /ج/ pronounced as the j in _Jim_by /ğ/ (instead of /j/): so the letter can be read easily not only in French or English but also in German, Spanish, Italian, etc.

To simplify reading, this variation can be used:
 /ت/ and  /ط/ → /t/
 /ث/ → /th/
 /ح/ and  /ه/ → /h/
 /خ/ → /kh/
/ذ/ and  /ض/ → /dh/
/ش/ → /sh/
 /غ/ → /gh/

For more, see [Romanization of Arabic](https://en.wikipedia.org/wiki/Romanization_of_Arabic).

Reply:
Excellent. I would prefer Laffite/Kunitzsch, as they are more accurate. Translators to other languages may then adopt their localized transliterations (hopefully in concordance with the single-names...)

Reply:
I will start this, but I expect it to take some time because of the absence of those letter shapes on the keyboard.

Reply:
Yes, I know this is likely slow work with "insert character" from some extended tables. But if made well, it will be a great and very valuable addition. Many thanks!